packages/manager/apps/dedicated/client/app/billing/billing.routing.js

```diff
@@ -14,8 +14,10 @@ export default /* @ngInject */ ($stateProvider) => {
           $state.transition.to().name !== 'app.account.billing.autorenew.ssh'
         ) {
           return $q.reject({
+            data: {
+              message: 'Access forbidden for enterprise accounts',
```

Comment:
Question, why we don't translate this message ?

Reply:
It was done that way as enterprise accounts are only available in the US